### PR TITLE
new instance type AcademicCommentary added, and improved logging on …

### PR DESCRIPTION
**Pull request by ketilaa**
…failure to create user

### CODE REVIEW

**Review by sondrev — APPROVED** [review]

File: user-access-handlers/src/main/java/no/unit/nva/handlers/CreateUserHandler.java [hunk 1]
```diff
@@ -232,7 +232,7 @@ public static ApiGatewayException castToCorrectRuntimeException(Failure<?> failu
         if (exception instanceof ConflictException) {
             return (ConflictException) exception;
         } else {
-            LOGGER.error(failure.getException().getMessage());
+            LOGGER.error("Failed to create or update user!", failure.getException());
```
Comment (torarnet) [inline user-access-handlers/src/main/java/no/unit/nva/handlers/CreateUserHandler.java:235]: We want the whole stacktrace, not just the message?

Comment (torarnet) [reply user-access-handlers/src/main/java/no/unit/nva/handlers/CreateUserHandler.java:235]: Or does it gets formatted by the logger?

Comment (ketilaa) [reply user-access-handlers/src/main/java/no/unit/nva/handlers/CreateUserHandler.java:235]: We want the source of the problem hidden in the stacktrace, yes. Currently it logs this:

ERROR - Expected a single value, but 0 were found

**Review by torarnet — APPROVED** [review]
Seems codacy complains, but approved otherwise

**Pull request merged**